packages/projection/src/operators/Mappers/certificates/withStakeKeyRegistrations.ts

```diff
@@ -19,10 +19,15 @@ export const withStakeKeyRegistrations = unifiedProjectorOperator<WithCertificat
     ...evt,
     stakeKeyRegistrations: evt.certificates
       .map(({ pointer, certificate }): StakeKeyRegistration | null => {
-        if (certificate.__typename === Cardano.CertificateType.StakeRegistration) {
+        if (
```

Comment:
An helper as `stakeKeyCertificates` could help increasing the readability... making it a type checker would also help removing the `as Cardano.RegAndDeregCertificateUnion` a few lines later.